scss/_reboot.scss

```diff
@@ -411,6 +411,15 @@ button {
   border-radius: 0;
 }
 
+// Explicitly remove focus outline in Chromium when it shouldn't be
+// visible (e.g. as result of mouse click or touch tap). It already
+// should be doing this automatically, but seems to currently be
+// confused and applies its very visible two-tone outline anyway.
+
+button:focus:not(:focus-visible) {
+  outline: 0;
```

Comment:
We have one instance in _close.scss where we use `none`. Not sure if that's intentional, and it's unrelated to this PR but I noticed it and we might want to be consistent (in another PR).

Reply:
good call, yes. i'll get onto that later today